operator/pkg/ciliumendpointbatch/endpointbatch.go

```diff
@@ -156,7 +166,8 @@ func (c *CiliumEndpointBatchController) Run(ces cache.Indexer, stopCh chan struc
 	log.WithFields(logrus.Fields{
 		"Upsert syncperiod": c.ciliumEndpointBatchUpsertSyncPeriod,
```

Comment:
With workqueue, we no longer have these two sync periods, correct?

Reply:
Thats right, we aren't using sync period anymore. I would keep it around until we are sure we don't need anymore.

Reply:
If they are not used, I think we should remove them. We can add back in the future if needed.

Reply:
SG to me.